CHANGELOG.md

```diff
@@ -12,6 +12,41 @@ Bugfixes:
 
 Other improvements:
 
+## [v5.0.0](https://github.com/purescript/purescript-prelude/releases/tag/v4.1.0) - 2021-MONTH-DATE
```

Comment:
Oh, last thing, maybe it would be good to leave this stuff under "Unreleased" for now so that we don't forget to update the placeholder link and date? I don't really mind though.

Reply:
Makes sense to me. I'll add that change and then merge.